compiler-rt/lib/dfsan/dfsan.cpp

```diff
@@ -519,7 +606,10 @@ static void dfsan_init(int argc, char **argv, char **envp) {
   Atexit(dfsan_fini);
   AddDieCallback(dfsan_fini);
 
-  __dfsan_label_info[kInitializingLabel].desc = "<init label>";
+  //label_map_t::Handle new_label(&__dfsan_label_info, kInitializingLabel);
```

Comment:
Why is this commented out?

Reply:
This was my use of the hashtable I modified from the sanitizer lib, and was using it earlier. Instead, we are pushing this functionality into polytracker. So we no longer need it, its basically just dead code, we can remove it